sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -571,7 +588,7 @@ export class MessageSender extends LinkEntity {
         this.name,
         data
       );
-      return await this._trySend(encodedMessage);
+      return await this._trySend(encodedMessage, false);
```

Comment:
This change is not needed as the second argument defaults to falsy value

Reply:
Had to add this when I was passing the Options. Removed now.